frontends/web/src/routes/account/add/add.tsx

```diff
@@ -237,7 +237,7 @@ export const AddAccount = ({ accounts }: TAddAccountGuide) => {
                   </Step>
                 </Steps>
               </div>
-              <div className="row flex flex-row flex-between m-bottom" style={{ flexDirection: 'row-reverse' }}>
+              <div className={styles.buttonsContainer} style={{ flexDirection: 'row-reverse' }}>
```

Comment:
I'd slightly prefer to refactor this instead with View/ViewButtons components, but I guess the layout wouldn't look  100% same.